test/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../.eslintrc",
```

Comment:
http://eslint.org/docs/user-guide/configuring.html#configuration-cascading-and-hierarchy you are correct...